application/common/components/MfaBackendManager.php

```diff
@@ -103,11 +104,18 @@ public function verify(int $mfaId, string $value, string $rpOrigin = ''): bool
     /**
      * Delete MFA backend configuration
      * @param int $mfaId
+     * @param string $credId Credential ID (only used for WebAuthn)
      * @return bool
      * @throws ServerErrorHttpException
      */
-    public function delete(int $mfaId): bool
+    public function delete(int $mfaId, string $credId=''): bool
     {
+        if ($credId != '') {
+            throw new ForbiddenHttpException(
+                sprintf("May not delete a credential on a %s mfa type", Mfa::TYPE_MANAGER),
```

Comment:
Does `$this->type` work instead of `Mfa:...`?

Reply:
I don't think so.  This is a different class than the Mfa class. It doesn't have a type attribute.